proposals/lip-0043.md

```diff
@@ -429,6 +432,9 @@ def verify(trs: Transaction) -> None:
     # Validator keys must be in lexicographic order.
     if not all(validatorKeys[i] < validatorKeys[i + 1] for i in range(len(validatorKeys) - 1)):
         raise Exception("Validator keys are not in lexicographic order.")
+    
+    if len(trs.params.mainchainValidators) < 1 or len(trs.params.mainchainValidators) > MAX_NUM_VALIDATORS:
```

Comment:
Here we can still check that `len(trs.params.mainchainValidators) > MAINCHAIN_NUMBER_ACTIVE_DELEGATES`, so that if 102 active delegates are registered, we raise an exception (with the current check we raise exception only in case the number of validators is more than 199)